app/models/group/sektions_funktionaere.rb

```diff
@@ -9,6 +9,8 @@ class Group::SektionsFunktionaere < ::Group
 
   self.static_name = true
 
+  validates :type, uniqueness: { scope: :parent_id }
```

Comment:
Was machen diese Changes die mit `static_name` zu tun haben hier? Gehören die nicht in einen anderen PR? Müsstest du noch auf master rebasen?

Reply:
:+1: merci, genau, die braucht es hier nicht mehr weil wir das jetzt im Cöre gelöst haben